modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+You can use the Helm CLI to install {product} on OpenShift or Kubernetes.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has the `developer` or `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. For a Kubernetes cluster, you  must first add the OpenShift Helm Charts repository. On Openshift, this is already present. 
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
```

Comment:
```suggestion
. Create an {ocp-short} or Kubernetes project:
```
Kubernetes == OpenShift - there's no reason why you'd create a project in one or a namespace in the other AFAIK, so I'd stick with creating a project here.

Reply:
Split in 2 procedures for clarity. The OpenShift procedure mentions project. The Kubernetes procedure mentions namespace.